documentation/sphinx/source/backups.rst

```diff
@@ -524,6 +524,12 @@ The ``start`` command will start a new restore on the specified (or default) tag
 ``--inconsistent-snapshot-only``
   Ignore mutation log files during the restore to speedup the process. Because only range files are restored, this option gives an inconsistent snapshot in most cases and is not recommended to use.
 
+``--user_data``
+  Restore only the user keyspace. This option should NOT be used alongside --system_metadata (below) and CANNOT be used alongside other specified key ranges
+
+``--system_data``
+  Restore only the relevant system keyspace (currently only tenant metadata). This option should NOT be used alongside --user_data (above) and CANNOT be used alongside other specified key ranges
```

Comment:
`currently only tenant metadata` should be removed, as per the command, it will restore all mutations belonging to the `system keyspace` i.e. it doesn't look for `tenant subkeyspace` and restore it. However, the default FDB backup behavior is, backup: TenantMap + user_space mutations, hence, only `TenantMap` gets restored. But, if cluster backup is configured with more systemspace key-ranges to be backed up, then, the command should restore it as well?

Reply:
Right now we use: `getSystemBackupRanges()` to get the system key restore ranges, which is the same function we use during backup.  So if more system key ranges are added to this function they will also be included in the restore

Reply:
Precisely my point; hence, we should remove `currently only tenant metadata` as it might not be needed or accurate with time. Thoughts?